packages/swap/src/event-handlers/swapDepositAddressReady.ts

```diff
@@ -1,9 +1,10 @@
-import { swappingSwapDepositAddressReady } from '@chainflip/processor/160/swapping/swapDepositAddressReady';
+import { swappingSwapDepositAddressReady as schema160 } from '@chainflip/processor/160/swapping/swapDepositAddressReady';
+import { swappingSwapDepositAddressReady as schema180 } from '@chainflip/processor/180/swapping/swapDepositAddressReady';
 import { z } from 'zod';
 import { calculateExpiryTime } from '../utils/function';
 import { EventHandlerArgs } from './index';
 
-const swapDepositAddressReadyArgs = swappingSwapDepositAddressReady;
+const swapDepositAddressReadyArgs = z.union([schema180, schema160]);
```

Comment:
forgot what changed here again 😅 but i think we dont need to handle?

Reply:
an enum value was added or removed I think. might not have to handle it in the logic, but we don't want the parsing to fail